Appium/generate_config.sh

```diff
@@ -7,7 +7,7 @@ if [ -z "$PLATFORM_NAME" ]; then
 fi
 
 if [ -z "$APPIUM_HOST" ]; then
-  APPIUM_HOST="127.0.0.1"
+  APPIUM_HOST=$(hostname -i)
```

Comment:
> -i, --ip-address
>     Display the IP address(es) of the host. Note that this works only if the host name can be resolved. Avoid using this option; use hostname --all-ip-addresses instead. 

Is it safe?

Reply:
https://stackoverflow.com/questions/13322485/how-to-get-the-primary-ip-address-of-the-local-machine-on-linux-and-os-x

Reply:
The script will run inside container and It will give the IP address of that running container so I think it should be fine.

Reply:
I think it should be fine